pages/build/back-end/micro-services.mdx

```diff
@@ -0,0 +1,143 @@
+
+## Usage
+The generated microservice consists of three main elements:
+<OrderedList>
+<ListItem>A versatile RDS configuration that supports PostgreSQL, MySQL, SQLite, MongoDB, CockRoachDB, and Microsoft SQL Server</ListItem>
+ <ListItem>A ready-to-use GraphQL API template</ListItem>
+ <ListItem>An easy deployment method to AWS AppSync.</ListItem>
+</OrderedList>
+
+```bash copy filename="terminal"
+vastly generate service
+```
+Run the `vastly generate service` command in the root of your project. A `services` directory will be populated with your new microservice along with a few tools for your backend architecture. While we provide tools that are somewhat opinionated, you are more than welcome to customize your backend architecture.
+
+### Folder Structure
+<Tabs items={["services folder", "apps folder"]}>
+<Tab>
+```bash {1-8}
+serviceName/
+  |__ handler.ts  # lambda function handler (API resolver)
+  |__ server.ts   # local server (for dev)
+  |__ cdk/        # AWS CDK deploy boilerplate
+  |__ prisma/
+    |__ schema.prisma  # prisma schema (data source)
+    |__ generated/     # auto-generated after each `npx prisma generate`
+  |_ package.json
+|__ README.md
+```
+</Tab>
+<Tab>
+```bash {2, 5-6}
+client/
+  |__ graphql/  # auto-generated api (queries, mutations, types) after each `generate` script
+  |__ pages/  # front-end application
+  |__ .eslintrc.json
+  |__ apollo.tsx  # configuration template for connecting your api to AWS AppSync (Apollo Client)
+  |__ codegen.ts  # configuration file for graphql code generation
+  |__ next.config.js
+  |__ package.json
+  |__ tsconfig.json
+```
+</Tab>
+</Tabs>
+### Tools
+By default, you'll be provided with Prisma.io and it's extension, Prisma AppSync. Prisma.io is a type-safe database toolkit that simplifies access and management, while Prisma Appsync is a plugin that integrates Prisma with AWS AppSync.
+
+<SimpleGrid columns={[1, 2]} gap="4" pt="6">
+  <LinkCard
+    heading="Learn Prisma"
+    description="Begin here if you are completely unfamiliar with the Prisma ORM"
+    link="https://www.prisma.io/docs/concepts/overview/what-is-prisma"
+    isExternal
+  />
+  <LinkCard
+  heading="Learn Prisma Appsync"
+  description="Learn how Prisma Appsync seamlessly connects your API to AWS Appsync"
+  link="https://prisma-appsync.vercel.app/quick-start/getting-started.html"
+  isExternal
+/>
+</SimpleGrid>
+
+### Scripts
+#### <Text as='kbd' fontSize='xl'>generate</Text>
+Navigate into your microservice directory to run these scripts.
+After each `generate`, files inside `prisma/generated` will be auto-generated. Creates the client, schema (from `/schema.prisma`), resolvers, and models mapping
+  <Tabs items={['pnpm', 'npm', 'yarn']}>
+    <Tab>
+      ```bash copy filename="services/[serviceName]"
+      pnpm generate
```

Comment:
pnpm is not supported by prisma-appsync, so for mvp, we are enforcing only yarn or npm will be used.